.github/workflows/release.yaml

```diff
@@ -0,0 +1,83 @@
+name: "release"
+
+on:
+  release: {}
+  workflow_dispatch:
+    inputs:
+      release:
+        description: "the version to publish, if you just want to regen the changelog.md set 0.0.0"
+        required: true
+        type: string
+      check:
```

Comment:
Is there a use case of calling this action with check == false?

Reply:
Yes to get the release notes ready before the tag is actually pushed.